ruleset.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
what about renaming this to `phpcs-ruleset.xml` to make it more clear what the ruleset is about?

Reply:
Initially I did that, but the the default name is `ruleset.xml`. and if we follow that, other editor linters picks it up automatically. But having other names is also not a bad idea. we just have to specify the path to the xml explicitly.

I was thinking maybe lets put it inside the `bin` folder? and name it `phpcs-ruleset.xml`.
What do you think?